aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapter.java

```diff
@@ -579,6 +579,25 @@ public synchronized void terminate() throws DataStoreException {
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void clear(@NonNull Action onComplete,
+                      @NonNull Consumer<DataStoreException> onError) {
+        try {
+            terminate();
+            sqliteStorageHelper.deleteDatabaseFromDisk();
+            onComplete.call();
+        } catch (Exception exception) {
+            DataStoreException dataStoreException = new DataStoreException(
+                "Error while trying to delete database from local storage.",
```

Comment:
Maybe, for more clarity, `"Error while trying to clear data from the local DataStore storage"` ?

Reply:
Updated message.